distributions/Dist_sar653_1.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+from base_distribution import BaseDistribution
+
+class Dist_sar653_1(BaseDistribution):
+	def __init__(self):
+		self.f_max = 1
+		self.x_min = -1
+		self.x_max = 1
+
+
+	def pdf(self, x):
+		"""This is your PDF"""
+		return np.sqrt(1-x**2)
+
+	def mean(self):
+		"""This is the mean of the PDF"""
+		return 0.
+
+	def std(self):
+		"""This is the standard deviation of the pdf"""
+		return np.sqrt(.393)
```

Comment:
you should have an equation based on self.x_min and self.x_max derived using calculus. Where does .393 come from?